meta/documents/changelog_en.md

```diff
@@ -1,5 +1,14 @@
 # Release Notes for PAYONE
 
+## X.X.X
+### Added
+- Payment methods Klarna "Slice It" (Installments), Klarna "Pay Later" (Invoice), Klarna "Pay Now" (Direct Debit), Klarna "Pay Now" (Direct Bank Transfer).
```

Comment:
This change goes then nevertheless also in the version 2.3.2 online.

Reply:
Sorry, im not sure i understand the comment :S . I should add this text 

The following payment methods have been added:
Klarna "Slice It" (Installments),
Klarna "Pay Later" (Invoice),
Klarna "Pay Now" (Direct Debit),
Klarna "Pay Now" (Direct Bank Transfer)